docs/notebooks/low_level.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>nit: I think dataframe is typically used as one word in the context of pandas</p><p> </p><p>nit: "The only thing is happening when accessing any "flat" representation is a creation of a new repeating index." -&gt; "The only additional overhead when accessing a "flat" representation is the creation of a new repeating index."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/nested-pandas/pull/71/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>